examples/streaming.rb

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env ruby
+
+require 'unleash'
+require 'unleash/context'
+
+puts ">> START streaming.rb"
+
+@unleash = Unleash::Client.new(
+  url: 'https://app.unleash-hosted.com/demo/api',
```

Comment:
demo doesn't have streaming enabled so this needs to be changed to enterprise URL